opentelemetry-otlp/src/exporter/tonic/logs.rs

```diff
@@ -5,23 +5,23 @@ use opentelemetry_proto::tonic::collector::logs::v1::{
 };
 use opentelemetry_sdk::error::{OTelSdkError, OTelSdkResult};
 use opentelemetry_sdk::logs::{LogBatch, LogExporter};
+use tokio::sync::Mutex;
```

Comment:
how about importing `use std::sync::Mutex;` instead, then you can basically use the same code as in `metrics` for `shutdown()` and `export()` and it should work at least for me it does 😅

Reply:
https://github.com/open-telemetry/opentelemetry-rust/pull/2764#discussion_r1985994302

Reply:
https://github.com/open-telemetry/opentelemetry-rust/issues/2777 Will use this for further discussion. I am unblocking this PR by removing tonic's shutdown logic. (it was never called anyway!)